README.md

```diff
@@ -69,6 +69,10 @@ the benchmark's workflow is executed.
   $ kubectl create -n kubestone -f config/samples/perf_v1alpha1_iperf3.yaml
   iperf3.perf.kubestone.xridge.io/iperf3-sample created
   ```
+* Start fio benchmark by creating CR
```

Comment:
This comment is for line 67: Can you please include there that the sample benchmark is for iperf (to be consistent with this extension)?

Reply:
done